shop/src/main/java/ro/msg/learning/shop/service/OrderService.java

```diff
@@ -0,0 +1,86 @@
+package ro.msg.learning.shop.service;
+
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.dto.OrderDetailDTO;
+import ro.msg.learning.shop.model.*;
+import ro.msg.learning.shop.repository.*;
+import ro.msg.learning.shop.service.exceptions.NotFoundException;
+import ro.msg.learning.shop.service.strategy.LocationStrategy;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+@Service
+public class OrderService {
+
+    private final PlacedOrderRepository placedOrderRepository;
+
+    private final CustomerRepository customerRepository;
```

Comment:
The service from one entity should contain only its own repository. You should create for each entity that you use a different service with its own methods and use the service here.